md-dao/src/main/java/com/wipro/ats/bdre/md/dao/GeneralConfigDAO.java

```diff
@@ -15,12 +15,14 @@
 package com.wipro.ats.bdre.md.dao;
 
 import com.wipro.ats.bdre.exception.MetadataException;
+import com.wipro.ats.bdre.md.beans.ClusterInfo;
 import com.wipro.ats.bdre.md.dao.jpa.GeneralConfig;
 import com.wipro.ats.bdre.md.dao.jpa.GeneralConfigId;
 import org.apache.log4j.Logger;
 import org.hibernate.Criteria;
 import org.hibernate.Session;
 import org.hibernate.SessionFactory;
+import org.hibernate.criterion.MatchMode;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.hibernate.criterion.MatchMode'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AUselessImportCheck)